### ISSUE TITLE
Future of the project servo-gtk

### ISSUE THREAD

**Issue by okias** [antoyo/servo-gtk#2]
Hello @antoyo ,

do you plan to continue work on this project at some point? It sound really interesting to have Rust written engine with binding to GTK!

Since I'm new to Rust, I cannot guess how hard is to write simple browser a top of `servo-gtk`, but it would be nice to have alternative to webkit-gtk .

Thank you for info.

**Comment by antoyo** [comment]
Unfortunately no, at least not until Servo become more usable than the alternatives.

Also, I'm trying to make a CEF binding for GTK. I would have prefer Gecko, but it's not so easy to embed.

**Comment by matzipan** [comment]
Hi, 

Thanks for the awesome work. Could you elaborate on the difficulties you encountered? 

The thought of going back to developing with webkit2-gtk is making me sad.

**Comment by antoyo** [comment]
I meant that Servo is not mature: some websites won't work correctly in it.

Also, when I tried updating to a more recent servo, the webview got black and I wasn't sure how to fix it.

**Comment by okias** [comment]
@antoyo have you tried to chat with devs about it? https://chat.mozilla.org/#/room/#servo:mozilla.org

**Comment by matzipan** [comment]
That's quite interesting. I always thought Firefox was already using it. But I realize now Firefox is just using Stylo right now.

But that's reassuring, because I'm don't want to render complex web pages, just email rendering.

Thanks for the info

**Comment by antoyo** [comment]
If anybody wants to continue this project, my plan was to use [gnome-class](https://gitlab.gnome.org/federico/gnome-class) to make a shared library of servo-gtk (so that people don't have to compile servo every time) and make a Rust wrapper.

**Comment by john01dav** [comment]
Did you ever get CEF bindings or anything other than webkit2gtk working with gtk? I'd just use webkit2gtk, but it lacks Windows support, and I need Windows support.

**Comment by antoyo** [comment]
@john01dav: [That](https://github.com/dungeonfog/cef) might work for you on Windows. I have a problem with it on Linux.

**Comment by antoyo** [comment]
If anyone wants to continue this project, please contact me.

**Comment by okias** [comment]
Hello @antoyo

does anything changes with Servo being handed to Linux Foundation? If you're still interested in this project, there could possibility that they could support you - of course if you're still interested! :smile:

**Comment by antoyo** [comment]
I don't think that changes anything.

You might find [this](https://github.com/jdm/servo-embedding-example/blob/main/src/main_gtk.rs) to be a good alternative, though.
